pkgs/development/python-modules/tld/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchPypi, buildPythonPackage, six }:
+
+buildPythonPackage rec {
+  pname   = "tld";
+  version = "0.9.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0i0prgwrmm157h6fa5bx9wm0m70qq2nhzp743374a94p9s766rpp";
+  };
+
+  doCheck = false;
```

Comment:
can we run the tests?

Reply:
no, doesn't work. I asked on irc, one commit for that would be ok.

Reply:
You mean, the tld tests are broken upstream as well? If yes, please open an issue there and add a comment with a link to it next to this line.

Reply:
No they are required others python modules and i don't even use photon anymore ...

Reply:
So these modules would be a candidate for `checkInputs`.

Running unit tests for interpreted languages like python are important to catch breakages when dependencies are bumped, even more than with compiled languages.

Reply:
https://github.com/barseghyanartur/tld/tree/master/requirements too much works i look earlier. that failed on pytest-cov

Reply:
I enabled the tests, had to add the required testing libraries to `checkInputs`, and exclude some tests that seem to try to use networking.